addons/account/models/account_reconcile_model.py

```diff
@@ -24,7 +24,15 @@ def validate_regex(self):
         for record in self:
             if not (record.narration_regex or record.payment_ref_regex):
                 raise ValidationError(_("Please set at least one of the match texts to create a partner mapping."))
-
+            try:
+                if (record.payment_ref_regex):
+                    current_regex = record.payment_ref_regex
+                    re.compile(record.payment_ref_regex)
+                if(record.narration_regex):
```

Comment:
we don't usually put parenthesis when not necessary in python (and there is a missing space anyways)

Reply:
Yes, I apologize, I corrected it

Reply:
No need to apologize :slightly_smiling_face: